atd-etl/app/process/helpers_cr3.py

```diff
@@ -16,6 +16,7 @@
 import time
 from http.cookies import SimpleCookie
 
+import magic
```

Comment:
✨

Reply:
🪄

Reply:
I have always loved that `file` is backed by libmagic. You can even add your own magic in `/etc/magic` if you work with uncommon file types, but I've never had to do that.